spec/std/crystal/compiler_rt/divmod128_spec.cr

```diff
@@ -41,10 +41,10 @@ describe "__divti3" do
   test__divti3(2, -1, -2)
   test__divti3(-2, 1, -2)
   test__divti3(-2, -1, 2)
-  test__divti3(make_ti(-9223372036854775808, 0x0), 1, make_ti(-9223372036854775808, 0x0))
-  test__divti3(make_ti(-9223372036854775808, 0x0), -1, make_ti(-9223372036854775808, 0x0))
-  test__divti3(make_ti(-9223372036854775808, 0x0), -2, make_ti(0x4000000000000000, 0x0))
-  test__divti3(make_ti(-9223372036854775808, 0x0), 2, make_ti(-0x4000000000000000, 0x0))
+  test__divti3(0x80000000000000000000000000000000_u128.to_i128!, 1, 0x80000000000000000000000000000000_u128.to_i128!)
```

Comment:
Why `0x80000000000000000000000000000000_u128.to_i128!` and not `-0x80000000000000000000000000000000_i128`